### PR TITLE
chore: use glide

### PR DESCRIPTION
**Pull request by xtuc**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package: github.com/bytearena/bytearena
+ignore:
+- github.com/bytearena/bytearena
```
Comment (netgusto) [inline glide.yaml:3]: Fixes a self-referencing issue; without this, `glide install` tries to fetch bytearena packages (they're private, and local).

File: glide.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import:
+- package: github.com/bitly/go-notify
+- package: github.com/docker/distribution
+  version: f86db6b22663a27ba4d278220b7e34be528b1e79 # freezes master on 2017-07-04 23:54:45 Europe/Paris
```
Comment (netgusto) [inline glide.yaml:7]: Froze `github.com/docker/distribution` on today's master, as version `~2.6.1` misses this file : https://github.com/docker/distribution/blame/master/reference/normalize.go (which we use)

File: glide.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  subpackages:
+  - reference
+- package: github.com/docker/docker
+  version: 82c2a08b6018586f3c8f35405c3bd6fce3c78ccd # freezes master on 2017-07-04 23:54:45 Europe/Paris
```
Comment (netgusto) [inline glide.yaml:11]: Froze github.com/docker/docker on today's master, as version `~17.5.0-ce-rc3` changes a lot of stuff regarding to the master version we used so far.

Will be fixed when current master is released.

File: glide.yaml (hunk 1 above)
Comment (xtuc) [reply glide.yaml:3]: Didn't show up on my machine :(

**Pull request merged**